SplitFiles/control.c

```diff
@@ -114,7 +113,20 @@ void execute_command(ControlBuffer *ctrlBuff, GameParameters* params) {
             params->messageAddress = 100 + d - '0';
             BreadState* new_slice = malloc(sizeof(struct BreadStateStruct));
             new_slice->temperature = 0;
-            new_slice->moisture = ((e == 'B')? 75: 50) + rand()%50; //Brown takes longer
+            switch(e) {
```

Comment:
This is a great start - but long term I saw the bread types as being data-driven so they can be randomly ordered by customers and have different characteristics and scoring profile (ie harder to toast stuff gets ordered less often but scores more).